elliott/elliottlib/bzutil.py

```diff
@@ -18,7 +18,6 @@
 from errata_tool import Erratum
 from errata_tool.jira_issue import JiraIssue as ErrataJira
 from errata_tool.bug import Bug as ErrataBug
-from bugzilla.bug import Bug
```

Comment:
This was causing a conflict with the class definition, otherwise did not see it being used